core/templates/dev/head/directives.js

```diff
@@ -156,3 +156,19 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive( 'updateLanguage', updateLanguage );
+function updateLanguage( $rootScope ) {
+    return {
+      link: function( scope, element ) {
+        var listener = function( event, translationResp ) {
```

Comment:
translationResp - translation response .
I.e whenever a translation occurs the response is sent to this listener .